src/components/Button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Button({type, children }) {
```

Comment:
Dit component had van mij niet gehoeven. Het component is zo dunnetjes, dat het niet code reduceerd